dcos_launch/config.py

```diff
@@ -403,7 +406,7 @@ def _validate_genconf_scripts(genconf_dir, dcos_config):
         'type': 'string',
         'required': False,
         # bootstrap node requires docker to be installed
-        'default': 'cent-os-7-dcos-prereqs',
+        'default': 'coreos',
```

Comment:
Why change the default from `cent-os-7-dcos-prereqs` to `coreos`?

Reply:
Update: Charles and I talked about this. This change is only on 'bootstrap' node and we used `coreos` as it already had docker available.